boards/intel/wcl/Kconfig.defconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BUILD_OUTPUT_STRIPPED
+	default y
+
+config MP_MAX_NUM_CPUS
+	default 2
+
+#The APIC TSC deadline and one-shot timers on this board run at 2 GHz; the HPET runs at 19.2 MHz.
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 2000000000 if APIC_TSC_DEADLINE_TIMER
+	default 2000000000 if APIC_TIMER_TSC
+	default 19200000
+
+if APIC_TIMER
```

Comment:
Not formatted properly, see https://docs.zephyrproject.org/latest/contribute/style/kconfig.html